### ISSUE TITLE
removeChild  Error drag n drop

### ISSUE THREAD

**Issue by iliyaZelenko** [metafizzy/packery#332]
Sometimes when you drag n drop generates an error  

> Uncaught NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.

**Comment by desandro** [comment]
I'm sorry to see you're having trouble with Packery. Could you provide a **reduced test case**? See [Submitting Issues in the contributing guidelines](https://github.com/metafizzy/packery/blob/master/contributing.md#submitting-issues).

**Comment by iliyaZelenko** [comment]
[Link](http://codepen.io/anon/pen/OMGbZO). Try to press several times, then move any box, it's all you need to do quickly.

**Comment by desandro** [comment]
Thank you for reporting this bug! The bug has been identified and fixed in SHA: 05f1665c2ca5168b3e72a20c0da48bcb311bea23. The fix will be in the next version of Packery. In the meantime, you can add this code to resolve it:

``` js
Packery.Item.prototype.hideDropPlaceholder = function() {
  // only remove once, #333
  var parent = this.dropPlaceholder.parentNode;
  if ( parent ) {
    parent.removeChild( this.dropPlaceholder );
  }
}; 
```

**Comment by kubante** [comment]
Nice! this also fixes usage with react!  as a goodie, it would still be very neat if you could disable the placeholder in the options..

**Comment by desandro** [comment]
@kubante The placeholder is just an element. If you'd rather not see it, do not add any CSS styles to display it.

**Comment by desandro** [comment]
This fix was added in v2.1.0 back in April 2016.
